ykman/pcsc/__init__.py

```diff
@@ -139,7 +140,7 @@ def kill_scdaemon():
                 killed = True
     except ImportError:
         # Works for Linux and OS X.
-        return_code = subprocess.call(["/usr/bin/pkill", "-9", "scdaemon"])  # nosec
+        return_code = subprocess.call([shutil.which("pkill"), "-9", "scdaemon"])  # nosec
```

Comment:
This shouldn't even need `shutil`, right? Just `pkill` will make it fall back to `$PATH`.
```suggestion
        return_code = subprocess.call(["pkill", "-9", "scdaemon"])  # nosec
```

Reply:
Not super familiar with subprocess.call but this may well be the case.